manifests/package.pp

```diff
@@ -177,10 +177,17 @@
 
   if ($elasticsearch::package_provider == 'package') {
 
+    if $elasticsearch::manage_repo {
```

Comment:
This is not compatible with other distro's in its current form as it will even for none debian based distro's will do the require.